code/modules/atmospherics/machinery/portable/scrubber.dm

```diff
@@ -60,6 +60,11 @@
 	excited = TRUE
 
 	var/atom/target = holding || get_turf(src)
+	if(!holding)
+		var/turf/target_turf = target
+		for(var/turf/adjacent_turf in target_turf.get_atmos_adjacent_turfs())
+			scrub(adjacent_turf.return_air())
+			return ..()
```

Comment:
this will just scan one turf and then return parent

Reply:
tabbed one time too many